### PR TITLE
[HER-71] Create business command

### PR DESCRIPTION
**Pull request by andreea-dumitrascu**
## Description
- It is going to be a command inside the business layer
- It is going to be named CreateDetection
- It is going to have it`s own interface (ICreateDetction)
-  It is going to use the MCreateDetection as parameter
- It will return an MDetection
- Create dummy method in repository

## Related
[HER-71](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-71)

### CODE REVIEW

File: src/Business/Detection/Common/Repositories/IDetectionRepository.cs [hunk 1]
```diff
@@ -2,9 +2,12 @@ namespace Business.Detection.Common.Repositories
 {
     using System.Collections.Generic;
     using Business.Detection.Common.Models;
+    using Business.Detection.Creating.Models;
 
     public interface IDetectionRepository
     {
         public IEnumerable<MDetection> All();
+
+        public MDetection Create(MCreateDetection item);
```
Comment (danidoff) [inline src/Business/Detection/Common/Repositories/IDetectionRepository.cs:11]: Should we change this method to insert a List of MCreateDetection input and receive a List of MDetection? It should look like this: `public IEnumerable<MDetection> Insert(IEnumerable<MCreateDetection> input)`

Comment (danidoff) [reply src/Business/Detection/Common/Repositories/IDetectionRepository.cs:11]: Let`s have a talk about this.

**Review by danidoff — CHANGES_REQUESTED** [review]

File: src/Business/Detection/Common/Repositories/IDetectionRepository.cs (hunk 1 above)
Comment (andreea-dumitrascu) [reply src/Business/Detection/Common/Repositories/IDetectionRepository.cs]: Done.

File: src/Business/Detection/Common/Commands/ICommand.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-namespace Business.Detection.Fetching.Commands
+namespace Business.Detection.Common.Commands
```
Comment (RaduBuciuceanu) [inline src/Business/Detection/Common/Commands/ICommand.cs:1]: As far as this command is gonna be used for any further features we are going to have, won't be better to move it under the root `Business` project (I mean at the root level of the `Business` project)?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Business/Detection/Common/Commands/ICommand.cs (hunk 2 above)
Comment (andreea-dumitrascu) [reply src/Business/Detection/Common/Commands/ICommand.cs]: Done.

**Review by RaduBuciuceanu — APPROVED** [review]

**Review by danidoff — DISMISSED** [review]

**Review by danidoff — DISMISSED** [review]

**Review by ConstantinescuAndrei — DISMISSED** [review]

**Review by Urzica-Platon-Casian — DISMISSED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]
Sexy.

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**